ldaptor/test/test_server.py

```diff
@@ -729,6 +721,52 @@ def onCommit_(result, info):
             salt = raw[20:]
             self.assertEqual(entry.sshaDigest(b'hushhush', salt), secret)
 
+    def test_passwordModify_someoneElse(self):
```

Comment:
Please add a docstring to this tests.

It should explain that is the scope / target of this test.